test_pydantic_schemas.py

```
@@ -0,0 +1,409 @@
+        print(f"   ✅ Serialization: {len(json_data)} fields")
+        
+        # Test deserialization
+        audit_from_json = DataCollectionAuditCreate(**json_data)
+        print(f"   ✅ Deserialization: {audit_from_json.data_type}")
+        
+        # Test Response schema with additional fields
+        response_data = {
+            **valid_data,
+            "id": str(uuid4()),
+            "operation_id": str(uuid4()),
+            "time": datetime.now(timezone.utc).isoformat(),
```

Ruff (0.12.2)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)